pipeline_readmes/README-GSC.md

```diff
@@ -1,60 +1,72 @@
+This file contains a description of the different output files of the gene set characterization (GSC) pipeline. The downloaded zip file will contain up to nine other reference files. The format of some output files will vary depending on the parameter options selected.
 
-#### Files in Gene Set Collection Directory:
+#### Results File
  - A) gsc_results.txt - Gene Set Characterization Results File
- - B) gene_set_name_map.txt - Public Gene Set Mapping File
- - C) run_params.yml - Run Parameters File
- - D) *.metadata - Gene Set Collection Metadata 
 
-#### Other Reference Files:
- - E) clean_genomic_matrix.txt - Mapped Genomic Spreadsheet File
- - F) gene_map.txt - Gene ID Mapping File
- - G) interaction_network.metadata - Knowledge Network Metadata (if KN guided analysis)
+#### Reference Files:
+ - B) clean_gene_set_matrix.txt - Mapped Genomic Spreadsheet File
+ - C) gene_map.txt - Gene ID Mapping File
+ - D) gene_map_exceptions.txt - Gene ID Mapping Exceptions File
+ - E) dropped_genes.txt - Dropped Genes File
+ - F) run_params.yml - Run Parameters File
+ - G) run_cleanup_params.yml - Cleanup Run Parameters File
+ - H) property_networks.metadata - Gene Set Collection Metadata 
+ - I) interaction_network.metadata - Knowledge Network Metadata (if KN guided analysis)
 
 Below are descriptions for the contents of each of these files:
 
 #### A) gsc_results.txt - Gene Set Characterization Results File
 - The columns of this file will depend on the choice you made whether to "Use the Knowledge Network"
 - If "Yes" Option (DRaWR):
   1) user_gene_set: the names of the gene sets that you submitted
-  2) property_gene_set: the internal KnowEnG ids for public gene sets from one public gene set collection
-  3) difference_score: difference of query_score (col4) and baseline_score(col5) divided by the largest difference in the file. This value is between 0 and 1 and reported when it is greater than 0.5
-  4) query_score: converged stationary probability of being at the property_gene_set node in the chosen heterogenous network given that you restart at any only gene node from the user gene set. This value is between 0 and 1.
-  5) baseline_score: converged stationary probability of being at the property_gene_set node in the chosen heterogenous network given that you restart at any gene node. This value is between 0 and 1.
+  2) property_gene_set_id: the internal KnowEnG ids for public gene sets from one public gene set collection
+  3) property_gene_set_alias: alias for the gene set from its original data source
+  4) property_gene_set_description: description for the gene set from its original data source
+  5) difference_score: difference of query_score (col4) and baseline_score(col5) divided by the largest difference in the file. This value is between 0 and 1 and reported when it is greater than 0.5
```

Comment:
now col6 and col7

Reply:
the reason the column changes is because you are inserting the alias and descriptions in your post-processed result, there were no relevant changes to the GSC image i think

Reply:
Thanks. Somebody's on the ball (you) and somebody's not (me). Will fix.